frontend/components/course/Tools.tsx

```diff
@@ -0,0 +1,44 @@
+import { useTranslation } from "next-i18next";
+import { FC, useCallback } from "react";
+
+import { Course_Course_by_pk } from "../../queries/__generated__/Course";
+import { Button } from "../common/Button";
+
+interface IProps {
+  course: Course_Course_by_pk;
+}
+
+export const Tools: FC<IProps> = ({ course }) => {
+  const { t } = useTranslation("course-page");
+
+  const openChat = useCallback(() => {
+    if (!course.LinkChat) {
+      return;
+    }
+
+    window.open(course.LinkChat);
+  }, [course.LinkChat]);
+
+  const openVideoCall = useCallback(() => {
+    if (!course.LinkVideoCall) {
+      return;
+    }
+
+    window.open(course.LinkVideoCall);
+  }, [course.LinkVideoCall]);
+
+  return (
+    <div className="flex flex-1 gap-x-4">
+      {course.LinkChat && (
+        <Button filled onClick={openChat}>
+          {t("openChat")}
+        </Button>
+      )}
+      {course.LinkVideoCall && (
+        <Button filled onClick={openVideoCall}>
```

Comment:
Isn't it always better to use a normal href-Browser-link instead of an onClick handler?
Or is there a special reason to use the onClick handler here?

Reply:
I don't know. What is the advantage of using an Anchor? Should we change this?

If yes: What would be your preferred solution? Should we add a `href` and `target` prop to the Button Component?  Should we create a new component looking the same as the button but renders a HTML Anchor?

BTW: Why is the Button component rendering a HTML-span not a HTML-button?